modules/bookClass.js

```diff
@@ -0,0 +1,8 @@
+// Book class
+export default class Book {
+  constructor(title, author) {
+    this.id = Date.now().toString();
+    this.title = title;
+    this.author = author;
+  }
```

Comment:
- [ ] Please, it's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions, Kindly make sure to address the requested changes :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)